python/pyspark/sql/readwriter.py

```diff
@@ -388,16 +388,18 @@ def csv(self, path, schema=None, sep=None, encoding=None, quote=None, escape=Non
         return self._df(self._jreader.csv(self._spark._sc._jvm.PythonUtils.toSeq(path)))
 
     @since(1.5)
-    def orc(self, path):
-        """Loads an ORC file, returning the result as a :class:`DataFrame`.
+    def orc(self, paths):
```

Comment:
Good catch, I should not break the compatibility.  BTW,  I found that `DataFrameReader.parquet` use variable length argument which is not consistent with other file formats such as text, json and orc that use string or a list of string.  I can fix this in this PR or can do it in another PR to make them consistent. What do you think ?

```
@since(1.4)
    def parquet(self, *paths):
        """Loads Parquet files, returning the result as a :class:`DataFrame`.
```